src/Core/Checkout/Document/DocumentConfiguration.php

```diff
@@ -4,6 +4,7 @@
 
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\Framework\Struct\Struct;
+use Shopware\Core\System\Country\CountryEntity;
 
 #[\AllowDynamicProperties]
 #[Package('after-sales')]
```

Comment:
you could add a `@codeCoverageIgnore` here, since it is only a struct, then all the annoying messages about missing coverage should be gone.